### PR TITLE
feat (build User model)

### PR DESCRIPTION
**Pull request by CW-MD**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: server/db/models/user.js [hunk 1]
```diff
@@ -26,6 +38,45 @@ const User = db.define('user', {
   },
   googleId: {
     type: Sequelize.STRING
+  },
+  street: {type: Sequelize.STRING},
+  city: {type: Sequelize.STRING},
+  state: {type: Sequelize.CHAR(2)},
+  postalCode: {type: Sequelize.FLOAT(5)},
```
Comment (gretad711) [inline server/db/models/user.js]: Should this be a float or an integer? Postal codes are whole numbers.

File: server/db/models/user.js [hunk 2]
```diff
@@ -26,6 +38,45 @@ const User = db.define('user', {
...
+  Country: {
+    type: Sequelize.STRING,
+    defaultValue: 'United States'
+  },
+  cardType: {
+    type: Sequelize.ENUM('visa', 'mastercard', 'AMEX'),
+    allowNull: false
+  },
+  cardNumber: {
+    type: Sequelize.FLOAT(16),
```
Comment (gretad711) [inline server/db/models/user.js]: Should this be a float?

**Review by gretad711 — APPROVED** [review]

File: server/db/models/user.js (hunk 2 above)
Comment (SL8onC) [reply server/db/models/user.js]: thank you, fixed

File: server/db/models/user.js (hunk 1 above)
Comment (SL8onC) [reply server/db/models/user.js]: fixed

**Review by gretad711 — APPROVED** [review]

**Pull request merged**